packages/flutter/lib/src/material/page.dart

```diff
@@ -42,6 +42,8 @@ class MaterialPageRoute<T> extends PageRoute<T> with MaterialRouteTransitionMixi
     super.fullscreenDialog,
     super.allowSnapshotting = true,
     super.barrierDismissible = false,
+    super.traversalEdgeBehavior,
+    super.traversalDirectionedEdgeBehavior,
```

Comment:
"Directioned" isn't really a word.  I'd prefer something like `directionalTraversalEdgeBehavior`. Here, and for all the other places "Directioned" is used.